static/automatic_updates.sh

```diff
@@ -32,6 +32,7 @@ In the next step you will be able to choose to proceed or exit."
 
 if [[ "yes" == $(ask_yes_or_no "Do you want to enable automatic updates?") ]]
 then
+    occ_command config:app:set updatenotification notify_groups --value="[]"
     touch $VMLOGS/update.log
```

Comment:
Please run the permissions script here to make sure the update.log permissions are correct.

Reply:
do you mean this one?
https://github.com/nextcloud/vm/blob/master/static/setup_secure_permissions_nextcloud.sh
I don't get completely why but okay...

Reply:
Nvm, I was thinking wrong. :)